frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -113,6 +158,9 @@ const TaskOverview: FC<{
     (hasPermission(role, Permission.TaskEdit) && task.createdByUser === userId);
   const tasksPage = `${paths.roadmapHome}/${roadmapId}${paths.roadmapRelative.tasks}`;
   const [patchTaskTrigger] = apiV2.usePatchTaskMutation();
+  const { data: users, isLoading } = apiV2.useGetRoadmapUsersQuery(
+    Number(roadmapId) ?? skipToken,
```

Comment:
```suggestion
   roadmapId ?? skipToken,
```

It's already a number (or undefined), and `Number(undefined) ?? 'something'` is `NaN`

Reply:
roadmapId was a string since it came from useParams for the route, I changed it to use a selector instead since roadmaprouter should set it into state anyway